### ISSUE TITLE
Using the tutorial

### ISSUE THREAD

**Issue by jepa** [eco-data-science/rmarkdown_R#3]
Hey @oharac this is a fantastic tutorial! can I borrow it for our studyGroup?

cheers!

**Comment by oharac** [comment]
of course!  it's a bit old so please feel free to update it as you see fit!

**Comment by jepa** [comment]
Cool, thx!
